src/main/java/uk/gov/hmcts/reform/ccd/test/stubs/service/token/KeyUtil.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.hmcts.reform.ccd.test.stubs.service.token;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.security.spec.X509EncodedKeySpec;
+
+public class KeyUtil {
+
+    public static PublicKey getPublicKey(String filename) {
+        try {
+            final byte[] keyBytes = Files.readAllBytes(Paths.get(filename));
+            final X509EncodedKeySpec spec = new X509EncodedKeySpec(keyBytes);
+            return factory().generatePublic(spec);
+        } catch (IOException ex) {
+            throw new RuntimeException(
+                "Error loading public key '" + filename + "'.", ex
+            );
+        } catch (final InvalidKeySpecException ex) {
+            throw new RuntimeException(
+                "Error reading public key '" + filename + "'.", ex
+            );
+        }
+    }
+
+    public static PrivateKey getPrivateKey(String filename) {
+        try {
+            final byte[] keyBytes = Files.readAllBytes(Paths.get(ClassLoader.getSystemResource(filename).toURI()));
```

Comment:
Docker has some issue reading files using  `getSystemResource`. Please have a look and test using ccd-docker management-web UI login